proto/batch_plan.proto

```diff
@@ -25,11 +25,6 @@ message RowSeqScanNode {
   // Whether the order on output columns should be preserved.
   bool ordered = 5;
 
-  message ChunkSize {
-    uint32 chunk_size = 1;
-  }
-  // If along with `batch_limit`, `chunk_size` will be set.
-  ChunkSize chunk_size = 6;
```

Comment:
Shall we just mark it as `deprecated`? So others won't accidentally use `6` next time.

Reply:
I guess simply removing it is okay. We never persist batch plans.

Reply:
I guess we can simply use `limit = 6` to replace `chunk_size =6`.